.rubocop.yml

```diff
@@ -1,3 +1,3 @@
 inherit_from:
-- https://raw.githubusercontent.com/ManageIQ/guides/master/.rubocop_base.yml
+- https://raw.githubusercontent.com/RedHatInsights/insights-api-common-rails/master/.rubocop_base.yml
```

Comment:
@lindgrenj6  Is this change the cause of the Code Climate failure?

Reply:
Not because of the code climate - it was for dippy-bot, that way it can run the rubocop rules successfully. Codeclimate won't quit yelling at us over the copied code of mixing-in various controllers.

Reply:
But the failure doesn't seem to be due to copied code. It's failing in setup:
```
codeclimate prepare
Failed fetching https://raw.githubusercontent.com/ManageIQ/guides/master/.rubocop_base.yml: code=404 body=404: Not Found
```
That's why I ask about that specific change.

Reply:
Oh - yeah codeclimate.yml will need to be updated too. I actually think I'll pull this change out and open a new PR with both of those changes in it.

Reply:
I'm also for separated PR